components/nimbus/ios/Nimbus/NimbusApi.swift

```diff
@@ -124,14 +124,23 @@ public extension Notification.Name {
 /// This struct is used during in the `create` method to point `Nimbus` at the given `RemoteSettings` server.
 ///
 public struct NimbusServerSettings {
-    let url: URL
+    public init(url: URL) {
+        self.url = url
+    }
```

Comment:
Same comment about getting the `init` for free from Swift, I don't understand why this is necessary if the struct is already declared public?